Sample_v3/README.md

```diff
@@ -0,0 +1,36 @@
+# 📲 Stream Chat Sample
+
+This folder contains the source code for Stream Chat's official sample app. You can use it to get a preview of the features and for other testing purposes. It contains multiple samples for different design patterns and use cases in one project using
+both **StreamChat** and **StreamChatUI** (coming soon).
+
+## 👩‍🏫 Instructions
+
+<img align="right" src="https://i.imgur.com/WpYeSGh.png" width="33%" />
+
+### 1. Installation
+
+To use the sample, clone this repository, run `carthage bootstrap` in the terminal, and open `StreamChat_v3.xcodeproj` on Xcode. Select the `Sample` target and run in your preferred device. To access every feature, use Xcode 12 and iOS 14. Due to a bug in carthage on Xcode 12, you may need to [use this workaround](https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323).
```

Comment:
I'm afraid this needs to be easier and more bulletproof. We should mention how to install Carthage + we should probably include the workaround script directly into the repo until it's fixed. So ideally, the step would be:
- clone the repo
- install Carthage (if needed)
- run `./sample-bootstrap.sh`